class-php-redis-purger.php

```diff
@@ -1,9 +1,20 @@
 <?php
+/**
+ * Class file for PHP_Redis_Purger
+ *
+ * @category Components
+ * @package WordPress
+ * @subpackage Redis_Full_Page_Cache_Purger
+ * @author Spirited Media <contact@spiritedmedia.com>
+ * @license MIT
+ * @link https://spiritedmedia.com
+ * @since 0.0.1
+ */
```

Comment:
I'm not sure all of this information is correct or entirely necessary but some kind of file-level comment is required for WordPress Coding Standards.